WritingDashboard/backend/app/__init__.py

```diff
@@ -19,7 +19,7 @@
 def create_app(config_class=Config):
     app = Flask(__name__)
     # Prevent CORS errors, make sure we can retrieve things from the react front-end without errors:
-    CORS(app, origins=['https://localhost', 'writingdashboard.xyz', 'https://writingdashboard.xyz'], expose_headers=["custom-filename"])
+    CORS(app, origins=['localhost'], expose_headers=["custom-filename"])
```

Comment:
is dit de bedoeling?

Reply:
Ja, is aangezien het achter een nginx reverse proxy zit vgm, en werkte in mijn tests iig...